src/components/input/input.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const Input = props => {
```

Comment:
No questions. Just wanted to say that it was really cool to see a suggestion list! Nice work!

Reply:
Yeah, I thought it would be cool to throw this in there so users can use it and select a city easily, and possibly save myself from any user typos :)